pkg/benchmarks/browser.js

```diff
@@ -71,10 +70,19 @@ $("#run").click(() => {
 
     Runner.run(benchmarks, runCount, options, {
 
-        readFile(file) {
+        readBenchmark(file) {
+            return $.ajax({
+                type: "GET",
+                url: `https://raw.githubusercontent.com/peggyjs/peggy/${BRANCH}/examples/${file}`,
+                dataType: "text",
+                async: false,
+            }).responseText;
+        },
+
+        readTest(file) {
             return $.ajax({
                 type: "GET",
-                url: `https://raw.githubusercontent.com/peggyjs/peggy/${BRANCH}/benchmark/${file}`,
+                url: `/development/benchmark-data/${file}`,
```

Comment:
as above, this should point at the local repo

the only thing hitting public URLs should be e2e tests, and these aren't the right URLs for those